PinpointKit/PinpointKit/Sources/Core/Editing/EditImageViewController.swift

```diff
@@ -632,14 +632,24 @@ public final class EditImageViewController: UIViewController, UIGestureRecognize
 }
 
 extension EditImageViewController: Editor {
-    public func setScreenshot(_ screenshot: UIImage) {
-        let oldScreenshot = imageView.image
-        
-        imageView.image = screenshot
-        
-        if screenshot != oldScreenshot {
-            clearAllAnnotations()
+
+    public var screenshot: UIImage? {
```

Comment:
We should document that setting this clears the annotations if the image is different